docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -0,0 +1,93 @@
+
+Getting Started with Galaxy
+===========================
+
+Galaxy workflow support in Dockstore is new and a preview of
+more full featured support to come.
+
+Unlike WDL and CWL, Galaxy
+workflows in the near term are primarily created and modified
+from the Galaxy workflow editor (GUI), instead of a text editor.
+
+Additionally Galaxy workflows cannot be launched by the Dockstore CLI,
+or launched via a 'Launch with' button on the workflow information page.
+
+Tutorial Goals
+--------------
+
+-  Learn about `Galaxy <https://training.galaxyproject.org/>`__
+-  Create and run a basic Galaxy workflow
+-  Export the workflow to a file
+-  Setup a GitHub account and repository
+-  Push the workflow to GitHub
+-  Make a GitHub release
+
+Create a basic Galaxy workflow
+------------------------------
+
+Create and run your workflow in Galaxy. Here  is a tutorial for `Creating, Editing, Importing Galaxy Workflows <https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-editor/tutorial.html>`__
+
+Export the workflow to a file
+-----------------------------
+
+In Galaxy:
+
+- Click on the Galaxy UI Workflow link at the top of the page.
+- Click on the workflow name to expose the drop down menu.
+- Click Download
+
+The exported JSON file with a '.ga' suffix describes the inputs,
+outputs, and Galaxy Tool Shed dependencies for your workflow.
+
+
+.. figure:: /assets/images/docs/galaxy_download.png
+   :alt: Download
+
+   Download
+
+Setting up GitHub
+-----------------
+
+You will need to add the Galaxy workflow file you downloaded to a source code
+repository that Dockstore knows about. GitHub is a good choice, and if you
+are not familiar with GitHub you can use this
+`tutorial <https://guides.github.com/activities/hello-world/>`__ to set up
```

Comment:
Can be an issue to follow-up with, this should really be an example Dockstore Galaxy workflow complete with .dockstore.yml rather than a generic github repo